### PR TITLE
Complete tasks in module_9

### PR DESCRIPTION
**Pull request by vladSHOKO**
@Lehersun module 9 ready to check

### CODE REVIEW

File: module_9/task1.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+<?php
+
+$line = 'Student, hello!';
+$control = [];
+$lenghtOfLine = strlen($line);
+
+for ($i = 1; $i <= $lenghtOfLine; $i++) {
+    $tmp = substr($line, strlen($line) - 1);
+    $line = substr($line, 0, -1);
+    $control[$tmp]++;
```
Comment (Lehersun) [inline module_9/task1.php]:
```
PHP Notice:  Undefined index: ! in C:\Users\user\Desktop\PHP\homework\module_9\task1.php on line 10
PHP Notice:  Undefined index: o in C:\Users\user\Desktop\PHP\homework\module_9\task1.php on line 10
PHP Notice:  Undefined index: l in C:\Users\user\Desktop\PHP\homework\module_9\task1.php on line 10
PHP Notice:  Undefined index: e in C:\Users\user\Desktop\PHP\homework\module_9\task1.php on line 10
PHP Notice:  Undefined index: h in C:\Users\user\Desktop\PHP\homework\module_9\task1.php on line 10
PHP Notice:  Undefined index:   in C:\Users\user\Desktop\PHP\homework\module_9\task1.php on line 10
PHP Notice:  Undefined index: , in C:\Users\user\Desktop\PHP\homework\module_9\task1.php on line 10
PHP Notice:  Undefined index: t in C:\Users\user\Desktop\PHP\homework\module_9\task1.php on line 10
PHP Notice:  Undefined index: n in C:\Users\user\Desktop\PHP\homework\module_9\task1.php on line 10
PHP Notice:  Undefined index: d in C:\Users\user\Desktop\PHP\homework\module_9\task1.php on line 10
PHP Notice:  Undefined index: u in C:\Users\user\Desktop\PHP\homework\module_9\task1.php on line 10
PHP Notice:  Undefined index: S in C:\Users\user\Desktop\PHP\homework\module_9\task1.php on line 10
```
I've got these errors during running your code

File: module_9/task1.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<pre>
+<?php
+
+$line = 'Student, hello!';
+$control = [];
+$lenghtOfLine = strlen($line);
+
```
Comment (Lehersun) [inline module_9/task1.php:4]: Pay attention at 
https://www.php.net/manual/en/function.str-split.php 
https://www.php.net/manual/ru/function.count-chars.php

File: module_9/task1.php [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+for ($i = 1; $i <= $lenghtOfLine; $i++) {
+    $tmp = substr($line, strlen($line) - 1);
+    $line = substr($line, 0, -1);
+    $control[$tmp]++;
+}
+
+var_dump($control);
```
Comment (Lehersun) [inline module_9/task1.php]: Don't forget about result view:
```
S - 1
t - 2
u - 1
d - 1
e - 2
n - 1
, - 1
 - 1
h - 1
l - 2
o - 1
! - 1

```

File: module_9/task3.php [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+$numbers = [];
+
+for ($i = 0; $i < rand(3, 20); $i++) {
+    array_unshift($numbers, rand(0, 10));
+}
+
+$sumOfNumbers = 0;
+foreach ($numbers as $key => $value) {
+    if ($key % 2 != 0) {
```
Comment (Lehersun) [inline module_9/task3.php]: `($key % 2)` and `($key % 2 != 0)` is the same

File: module_9/task3.php [hunk 5]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+$numbers = [];
+
+for ($i = 0; $i < rand(3, 20); $i++) {
```
Comment (Lehersun) [inline module_9/task3.php]: Be carefully with functions in comparitions with `for` function. Function will have been ran with every iteration and you will never have clear result. You should implement a variable before cicle and use variable in comparition

File: module_9/task4.php [hunk 6]
```diff
@@ -0,0 +1,23 @@
+<pre>
+<?php
+
+$city1 = 250;
+$city1Radius = 300;
+$city2 = 750;
+$city2Radius = 300;
```
Comment (Lehersun) [inline module_9/task4.php]:
```
На нём в случайных местах находятся два города. 
```

File: module_9/task4.php [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+for ($i = 0; $i < $countOfCars; $i++) {
+    $positionOfCar = rand(1, 1000);
+    $numberOfCar = $i + 1;
+    if (($positionOfCar < 250 or $positionOfCar > 400) and ($positionOfCar < 600
+            or $positionOfCar > 900)
+    ) {
+        echo "Машина {$numberOfCar} едет вне города на {$positionOfCar} со скоростью не более 70 километров ";
+    } else {
+        echo "Машина {$numberOfCar} едет по городу на {$positionOfCar} со скоростью не более 70 километров ";
```
Comment (Lehersun) [inline module_9/task4.php]: Speed of cars out of a city should be 90 km/h

File: module_9/task4.php [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+<?php
+
+$city1 = 250;
+$city1Radius = 300;
+$city2 = 750;
+$city2Radius = 300;
+
+$countOfCars = 10;//Количество машин
+
+for ($i = 0; $i < $countOfCars; $i++) {
```
Comment (Lehersun) [inline module_9/task4.php]: Please create an array with all cars and their positions before making a solution

File: module_9/task4.php [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+    if (($positionOfCar < 250 or $positionOfCar > 400) and ($positionOfCar < 600
+            or $positionOfCar > 900)
+    ) {
+        echo "Машина {$numberOfCar} едет вне города на {$positionOfCar} со скоростью не более 70 километров ";
+    } else {
+        echo "Машина {$numberOfCar} едет по городу на {$positionOfCar} со скоростью не более 70 километров ";
+    }
+}
+?>
+</pre>
```
Comment (Lehersun) [inline module_9/task4.php]: Do not use html tags here. This task is only about php.
If you want to make a result more pretty you could add `\n` in the en of the result string

File: module_9/task5.php [hunk 10]
```diff
@@ -0,0 +1,13 @@
+<?php
+
+$studentsCount = rand(1, 1000000);
```
Comment (Lehersun) [inline module_9/task5.php]:
```
$studentsCount = 11;

if ($studentsCount % 10 == 1) {
    echo "На занятии {$studentsCount} студент";
} elseif ($studentsCount % 10 >= 2 and $studentsCount % 10 <= 4) {
    echo "На занятии {$studentsCount} студента";
} elseif (($studentsCount % 10 >= 5 and $studentsCount % 10 <= 10)
    or $studentsCount % 100 >= 11 and $studentsCount % 10 <= 20
) {
    echo "На занятии {$studentsCount} студентов";
}
```

```
На занятии 11 студент

```

**Review by Lehersun — CHANGES_REQUESTED** [review]

File: module_9/task5.php (hunk 10 above)
Comment (Lehersun) [reply module_9/task5.php]: You also can resolve it using switch case
https://www.php.net/manual/ru/control-structures.switch.php

File: module_9/task3.php (hunk 4 above)
Comment (vladSHOKO) [reply module_9/task3.php]: ohhh exactly, 0 return false, 1-9 return true

**Review by vladSHOKO — COMMENTED** [review]
@Lehersun fix mistakes

**Review by vladSHOKO — COMMENTED** [review]
@Lehersun fix code style and mistakes.
 Check this commit please

File: module_9/task1.php (hunk 1 above)
Comment (vladSHOKO) [reply module_9/task1.php]: @Lehersun I dont understand the reason of these mistakes. Can i use function error_reporting() in same situations?

Comment (Lehersun) [reply module_9/task1.php]: It's because of  incrementing not set value in array $control[$tmp]++;
You have to make it another way

**Review by Lehersun — APPROVED** [review]

**Comment by Lehersun** [comment]
Good job! I like your solution for task number 5

**Pull request merged**